sale_commission/tests/test_sale_commission.py

```diff
@@ -319,6 +319,7 @@ def test_supplier_invoice(self):
             self.env["account.move"].with_context(default_type="in_invoice")
         )
         move_form.partner_id = self.partner
+        move_form.ref = "sale_comission_TEST"
```

Comment:
IMO ref field is not required in odoo core.

Reply:
Yes, it is for vendor bills.